Codecademy/client-modules
# fix(ProLabel): add title

### Overview


Adds a `title` tag to the `ProLabel` svg to fix an a11y bug on the catalog pages. The title is "Pro only" which matches what is currently read on the pro label for curriculum cards.


To test:
1. visit https://portal-app-pr-158.dev-eks.codecademy.com/catalog
2. open voice over and navigate via tab onto a career path card
3. confirm that it reads "pro only career path <path name> <difficulty level> <x lessons> ...."
4. navigate to a not pro course
5. confirm that it does not read "pro only"

### PR Checklist

- [x] Related to JIRA ticket: [REACH-1076](https://codecademy.atlassian.net/browse/REACH-1076)
